Sources/Widgets/Widgets3D/LineWidget/index.js

```diff
@@ -40,6 +41,9 @@ function vtkLineWidget(publicAPI, model) {
   shapeToRepresentation[
     handleRepresentationType.ARROW
   ] = vtkArrowHandleRepresentation;
+  shapeToRepresentation[
+    handleRepresentationType.GHOSTARROW
+  ] = vtkGhostArrowHandleRepresentation;
```

Comment:
What's the goal for a Ghost representation ? Why did you call it ghost ?

Reply:
this part of the code was deleted too